src/main/java/Bosco.java

```diff
@@ -0,0 +1,103 @@
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+public class Bosco {
+    private static final String DIVIDER = "\t____________________________________________________________";
+    private static final int MAX_TASKS = 100;
+
+    private static final Task[] tasks = new Task[MAX_TASKS];
+    private static int taskCount = 0;
+
+    private static void printWelcomeMessage() {
+        System.out.println(DIVIDER);
+        System.out.println("\t Hello! I'm Bosco APD." + System.lineSeparator() + "\t What can I do for you?");
+        System.out.println(DIVIDER);
+    }
+
+    private static void printExitMessage() {
+        System.out.println(DIVIDER);
+        System.out.println("\t Bye! Hope to see you again soon!");
+        System.out.println(DIVIDER);
+    }
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        String userInputString;
+        String[] words;
+        String description;
+        Task selectedTask;
+
+        printWelcomeMessage();
+
+        while (true) {
+            userInputString = in.nextLine();
+            words = userInputString.split("\\s+", 2);
+            String commandType = words[0];
+            if (commandType.equals("bye")) {
+                break;
+            }
```

Comment:
Any reason you place this conditional outside of the switch case, perhaps you can put it inside the switch case.

Reply:
This was from a past version that I have not updated to fit into the switch case block yet, I will attempt to do so in future iterations. Thanks!